README.md

```
@@ -74,40 +87,87 @@ pnpm db:push     # Apply migrations to the database
+
 - `GET /preferences` - Get user email preferences
 - `POST /preferences` - Update user email preferences
 - `GET /default-preferences` - Get default email preferences
 
 ### Email Sending
+
 - `POST /resend` - Resend the last sent email
 
+### Cron Jobs
+
+- `GET /cron/usage-reports` - Trigger usage reports cron job 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)